### PR TITLE
New corrections

### PR DESCRIPTION
**Pull request by krystian-kazmierczak**
I fixed some bugs: navigation, favicon, margin in Rating and displaying image in ProfilePage. 
It's not much, but I think it looks better now.
If you don't have suggestions then I would merge it in `develop`.

### CODE REVIEW

**Review by Kinga-parasiewicz — APPROVED** [review]
Very good job, maybe could you correct navigation?  we told about that

**Comment by krystian-kazmierczak** [comment]
I'm still working on it. 
I not quite like the look of it and I'm still looking good solution. 
I think part of the problem may be with the user's monitor displaying the website and we cannot fix this.

File: public/favicon.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<svg width="100" height="100" viewBox="0 0 40 40" fill="black" xmlns="http://www.w3.org/2000/svg">
-<rect width="100" height="100" x="0" y="0" fill="black" />
-<path d="M26.8774 24.9939L32.9858 27.4906C34.1458 27.9639 35.4158 27.1122 35.4158 25.8589V14.7939C35.4158 13.5406 34.1458 12.6872 32.9858 13.1606L26.8774 15.6572" stroke="white" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M10.5068 31.9014H20.9552C24.5952 31.9014 26.8785 29.5297 26.8785 26.1714V14.4797C26.8785 11.1214 24.6052 8.74969 20.9552 8.74969H10.5068C6.85516 8.74969 4.5835 11.1214 4.5835 14.4797V26.1714C4.5835 29.5297 6.85516 31.9014 10.5068 31.9014Z" stroke="white" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round"/>
+<svg width="40" height="40" viewBox="0 0 40 40" fill="transparent" xmlns="http://www.w3.org/2000/svg">
```
Comment (izabelanowak) [inline public/favicon.svg:1]: Icon looks much better now!

File: src/Navigation/Link/styled.js [hunk 2]
```diff
@@ -42,15 +42,14 @@ export const StyledNavLink = styled(NavLink).attrs(() => ({ activeClassName }))`
...
   border-radius: 24px;
   transition: 0.5s;
-  outline: none;
 
   &:hover {
     color: ${({ theme }) => theme.colors.black};
     background: ${({ theme }) => theme.colors.white};
   }
 
   &.${activeClassName} {
```
Comment (izabelanowak) [inline src/Navigation/Link/styled.js:51]: Now it looks better, but after pressing button border jumps strangely. It's hard to describe, I attach a gif showing what I mean.
outline

File: src/features/PeoplePage/index.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ const PeoplePage = () => {
                   key={person.id}
```
Comment (izabelanowak) [inline src/features/PeoplePage/index.js:54]: Images looks perfect now 😄

File: src/common/Rating/styled.js [hunk 4]
```diff
@@ -8,10 +8,10 @@ export const Wrapper = styled.div`
   justify-content: start;
   align-items: center;
   grid-gap: 8px;
-  margin: 0 0 12px 0;
+  margin: 12px 0;
```
Comment (izabelanowak) [inline src/common/Rating/styled.js:11]: I think we should add only margin-top to 12px and rest set to 0.

File: src/Navigation/Link/styled.js [hunk 5]
```diff
@@ -42,15 +42,14 @@ export const StyledNavLink = styled(NavLink).attrs(() => ({ activeClassName }))`
   padding: 14px 16px;
```
Comment (izabelanowak) [inline src/Navigation/Link/styled.js:42]: In this file I see one more error (we could forget about it before) there is missing curly bracket after
`@media (max-width: ${({ theme }) => theme.breakpoints.mobile}px) {
    padding: 0;
    gap: 8px;
`;`
please add it when you make corrections

File: src/common/Tile/styled.js [hunk 6]
```diff
@@ -77,28 +77,28 @@ export const Tile = styled.div`
 export const Image = styled.img`
```
Comment (izabelanowak) [inline src/common/Tile/styled.js:77]: If you changed height to auto in main `Image` style, you can delete other ` height: auto;` 😄 just a small note not to duplicate the same code 😃

**Review by izabelanowak — CHANGES_REQUESTED** [review]
I added a few comments from myself.
Great job 😃  but it still needs to be improved a bit.

**Review by izabelanowak — APPROVED** [review]
Ok, you can merge.

**Review by izabelanowak — APPROVED** [review]
Ok, you can merge.

**Pull request merged**